### DISCUSSION TITLE
suggestions about naming and structuring of an pentalogie

### DISCUSSION THREAD

**Discussion by skyzuma** [advplyr/audiobookshelf#1847] [General]
i need help for naming and structuring of a multi series serie like an pentalogie ... i have ["Stefan Burban - Das gefallene Imperium"](https://www.stefan-burban.de/das-gefallene-imperium) - sry for german website - and idk how i build the structure for a good and logical detection from ABS. There are 5 parts and if i see correct, its following:

part 1

- book 01
- book 02
- book 03
- book 04
- book 05

part 2
- book 06
- book 07
- book 08
- book 09
- book 10

part 3 - looks like a short story in this same universe / series
- book 1
- book 2
- book 3
- book 4
- book 5
- book 6

part 4 - is the continuation of part 2
- book 11
- book 12
- book 13
- book 14
- book 15

i dont like an series of any part ... atm i have this but i hate it ...

**Comment by MikeBishop** [comment]
There have been a couple of issues and discussions about subseries -- I also have several LONG series which are composed of multiple separate arcs.

Currently, the best model appears to be putting the series tags for both the mega-series and the sub-series.  Applying it to that series, I would use something like this:

- The Last bastion (The Fallen Empire #&#8288;1.1, The Drizil War #&#8288;1)
- The Battle of Vector Prime (The Fallen Empire #&#8288;1.2, The Drizil War #&#8288;2)
- ...
- Deceptive Peace (The Fallen Empire #&#8288;2.1, The Nefraltiri War #&#8288;1)
- Enemy Contact (The Fallen Empire #&#8288;2.2, The Nefraltiri War #&#8288;2)
- ...

Of course, that means you have both levels of series that show up in the series list unnecessarily, but it's the best current solution.  If you use the Collapse Sub-Series option, you can click on The Fallen Empire and see inside it The Drizil War, The Nefraltiri War, etc.

**Comment by skyzuma** [comment]
can u plz explain the structure in detail?
